packages/node/test/unit/node.spec.ts

```diff
@@ -1,12 +1,61 @@
+import * as ethers from "ethers";
+import firebase from "firebase";
+import firebasemock from "firebase-mock";
+
 import Node from "../../src/node";
+import {
+  Address,
+  AppInstanceInfo,
+  MethodName,
+  MethodRequest,
+  MethodResponse
+} from "../../src/node-types";
+
+describe("Node operations", () => {
+  const privateKey = process.env.npm_package_config_unlockedAccount0!;
+  const provider = ethers.getDefaultProvider() as ethers.providers.Web3Provider;
```

Comment:
I wouldn't use the default provider; that's the production etherscan geth instance I think. should assume localhost for tests